test/e2e/tests/privacy-mode/privacy-mode.spec.ts

```diff
@@ -0,0 +1,71 @@
+import { Driver } from '../../webdriver/driver';
+import {
+  withFixtures,
+  unlockWallet,
+  defaultGanacheOptions,
+} from '../../helpers';
+import FixtureBuilder from '../../fixture-builder';
+import HomePage from '../../page-objects/pages/home/homepage';
+import { loginWithBalanceValidation } from '../../page-objects/flows/login.flow';
+import { Ganache } from '../../seeder/ganache';
+import HeaderNavbar from '../../page-objects/pages/header-navbar';
+
+describe('Privacy Mode', function () {
+  it('should hide fiat balance and token balance when privacy mode is activated', async function () {
+    await withFixtures(
+      {
+        fixtures: new FixtureBuilder().build(),
+        ganacheOptions: defaultGanacheOptions,
+        title: this.test?.fullTitle(),
+      },
+      async ({
+        driver,
+        ganacheServer,
+      }: {
+        driver: Driver;
+        ganacheServer: Ganache;
+      }) => {
+        await loginWithBalanceValidation(driver, ganacheServer);
```

Comment:
when we login with balance validation we already ensure that the main balance element is displayed from start